dockerfiles/examples/local/hpcc-localfile/templates/dllstorage-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ printf "dll-%s-pvc" (include "hpcc-localfile.fullname" $) }}
+spec:
+  storageClassName: "manual"
+  volumeName: {{ printf "dll-%s-pv" (include "hpcc-localfile.fullname" $) }}
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ .Values.dllStorage.size }}
```

Comment:
I wonder whether should strictly be quoted as value exected to be alphanumeric, e.g. 2Gi

Reply:
quotes are not needed for alphanumeric in yaml (unless you want to include punctuation). Whether they make it clearer is debatable.

Reply:
Actually the schema is wrong - it should be string or numeric.